asterix/cat_000.ksy

```diff
@@ -0,0 +1,9 @@
+meta:
+  id: cat_000
```

Comment:
I guess more human-readable names are needed

Reply:
All names reply how they are referenced in the standards and in jargon, as much as Kaitai permits. Other names would be difficult to be correctly referenced by an hipothetical user. It is supposed if you use this files you already read the standards and know what the meaning of block, fspec CAT030 or I020/290 is.